<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DemoModeDialog</class>
 <widget class="QDialog" name="DemoModeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>388</width>
    <height>113</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Demo mode</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="labelROMStatesToSelect">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>ROM state filter</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QToolButton" name="toolButtonSelectC">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Select ROM state C (correct)?</string>
       </property>
       <property name="icon">
        <iconset resource="qmc2.qrc">
         <normaloff>:/data/img/sphere_green.png</normaloff>:/data/img/sphere_green.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonSelectM">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Select ROM state M (mostly correct)?</string>
       </property>
       <property name="icon">
        <iconset resource="qmc2.qrc">
         <normaloff>:/data/img/sphere_yellowgreen.png</normaloff>:/data/img/sphere_yellowgreen.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonSelectI">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Select ROM state I (incorrect)?</string>
       </property>
       <property name="icon">
        <iconset resource="qmc2.qrc">
         <normaloff>:/data/img/sphere_red.png</normaloff>:/data/img/sphere_red.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonSelectN">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Select ROM state N (not found)?</string>
       </property>
       <property name="icon">
        <iconset resource="qmc2.qrc">
         <normaloff>:/data/img/sphere_grey.png</normaloff>:/data/img/sphere_grey.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonSelectU">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Select ROM state U (unknown)?</string>
       </property>
       <property name="icon">
        <iconset resource="qmc2.qrc">
         <normaloff>:/data/img/sphere_blue.png</normaloff>:/data/img/sphere_blue.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelSecondsToRun">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Seconds to run</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="spinBoxSecondsToRun">
     <property name="toolTip">
      <string>Number of seconds to run an emulator in demo mode</string>
     </property>
     <property name="maximum">
      <number>999999</number>
     </property>
     <property name="value">
      <number>60</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="4">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="checkBoxFullScreen">
       <property name="toolTip">
        <string>Start emulators in full screen mode (otherwise use windowed mode)</string>
       </property>
       <property name="text">
        <string>Full screen</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxMaximized">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Maximize emulators when in windowed mode</string>
       </property>
       <property name="text">
        <string>Maximized</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxEmbedded">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Embed windowed emulators</string>
       </property>
       <property name="text">
        <string>Embedded</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxTagged">
       <property name="toolTip">
        <string>Use only tagged games</string>
       </property>
       <property name="text">
        <string>Tagged</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="4">
    <widget class="QPushButton" name="pushButtonClose">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Close this dialog (and stop running demo)</string>
     </property>
     <property name="text">
      <string>&amp;Close</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="5">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>324</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="4" rowspan="3">
    <widget class="QPushButton" name="pushButtonRunDemo">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Run demo now</string>
     </property>
     <property name="text">
      <string>Run &amp;demo</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="5">
    <widget class="QLabel" name="labelDemoStatus">
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="labelPause">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Pause (seconds)</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QSpinBox" name="spinBoxPauseSeconds">
     <property name="toolTip">
      <string>Number of seconds to pause between emulator runs</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
     <property name="value">
      <number>2</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>toolButtonSelectC</tabstop>
  <tabstop>toolButtonSelectM</tabstop>
  <tabstop>toolButtonSelectI</tabstop>
  <tabstop>toolButtonSelectN</tabstop>
  <tabstop>toolButtonSelectU</tabstop>
  <tabstop>spinBoxSecondsToRun</tabstop>
  <tabstop>spinBoxPauseSeconds</tabstop>
  <tabstop>checkBoxFullScreen</tabstop>
  <tabstop>checkBoxMaximized</tabstop>
  <tabstop>checkBoxEmbedded</tabstop>
  <tabstop>checkBoxTagged</tabstop>
  <tabstop>pushButtonRunDemo</tabstop>
  <tabstop>pushButtonClose</tabstop>
 </tabstops>
 <resources>
  <include location="qmc2.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>checkBoxFullScreen</sender>
   <signal>toggled(bool)</signal>
   <receiver>checkBoxMaximized</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>161</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonClose</sender>
   <signal>clicked()</signal>
   <receiver>DemoModeDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>383</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>292</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxFullScreen</sender>
   <signal>toggled(bool)</signal>
   <receiver>checkBoxEmbedded</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>240</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxTagged</sender>
   <signal>toggled(bool)</signal>
   <receiver>toolButtonSelectC</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>291</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>121</x>
     <y>24</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxTagged</sender>
   <signal>toggled(bool)</signal>
   <receiver>toolButtonSelectM</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>272</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>158</x>
     <y>23</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxTagged</sender>
   <signal>toggled(bool)</signal>
   <receiver>toolButtonSelectI</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>268</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>27</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxTagged</sender>
   <signal>toggled(bool)</signal>
   <receiver>toolButtonSelectN</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>268</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>6</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxTagged</sender>
   <signal>toggled(bool)</signal>
   <receiver>toolButtonSelectU</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>284</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>282</x>
     <y>25</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
